samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -33,6 +33,24 @@
 
 static const double meanDivisor = 100;
 
+static CGImageRef _CGImageCreate32bppCopy(CGImageRef image) {
```

Comment:
Given that this doesnt address the format conversion issues, i'd rather not fix this, and have an open issue to put in the real fix, which indeed provides the pixels in the proper expected format.